ros/api/v0/hosts.py

```diff
@@ -102,6 +111,48 @@ def get(self):
             limit, offset, hosts, count
         )
 
+    @staticmethod
+    def sorting_order(order_how):
+        """Sorting order method."""
+        if order_how == 'asc':
+            method_name = asc
+        elif order_how == 'desc':
+            method_name = desc
+        else:
+            abort(
+                403,
+                message="Incorrect sorting order. Possible values - ASC/DESC"
+            )
+        return method_name
+
+    def build_sort_expression(self, order_how, order_method):
+        """Build sort expression."""
+        sort_order = self.sorting_order(order_how)
+
+        if order_method == 'display_name':
+            return (sort_order(System.display_name),
+                    asc(PerformanceProfile.system_id),)
+        if order_method == 'cpu_score':
+            return (
+                sort_order(PerformanceProfile.performance_score[
+                    'cpu_score'].astext.cast(Integer)),
+                asc(PerformanceProfile.system_id)
+            )
+        if order_method == 'memory_score':
+            return (
+                sort_order(PerformanceProfile.performance_score[
+                    'memory_score'].astext.cast(Integer)),
+                asc(PerformanceProfile.system_id), )
+        if order_method == 'io_score':
+            return (
+                sort_order(PerformanceProfile.performance_score[
+                    'io_score'].astext.cast(Integer)),
+                asc(PerformanceProfile.system_id), )
+        # FIXME: no ordering as of now for columns:
```

Comment:
For the scores as order_methods, can we do something like:
```python
score_methods = ['memory_score', 'cpu_score', 'io_score']
if order_method in score_methods:
    return return (
                sort_order(PerformanceProfile.performance_score[
                    order_method].astext.cast(Integer)),
                asc(PerformanceProfile.system_id)
            )
```

Reply:
Good Catch :+1: